src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -117,7 +121,15 @@ const BioAndChat = (props) => {
         ) : (
           <Pressable onPress={openBio}>
             <Text linkStyle={styles.seeMore} style={styles.bioText}>
-              {bio}
+              {sanitizeNewLineBio?.map((bioLine, index) => {
+                if (index < sanitizeNewLineBio?.length - 1) {
+                  bioLine += '\n';
+                }
+
+                if (isValidUrl(bioLine)) return <LinkableText text={bioLine} />;
+
+                return <Text key={`${bioLine}-${index}`}>{bioLine}</Text>;
+              })}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for handling URLs in the bio text seems fine, but there's a potential issue with the key used for each `Text` component. If the same line of bio text appears more than once, React will throw a warning about duplicate keys. To avoid this, you could use the index as part of the key.
```diff
- return <Text key={`${bioLine}-${index}`}>{bioLine}</Text>;
+ return <Text key={`bio-${index}`}>{bioLine}</Text>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->